### ISSUE TITLE
[Accessibility][Advisory][Commerce Navigation] Menu option names are announced twice by NVDA screen reader.

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1791]
**Steps to Reproduce**
1. Enable NVDA screen reader.
2. Open page in Google Chrome.
3. Use TAB or TAB + SHIFT to navigate to the menu dialog trigger.
4. Activate menu dialog.
5. Navigate through the menu options.

**Current Behavior**
Because of an unnecessary `title` attribute on the menu option links (`<a>` elements) in addition to inline text, the names are announced twice.

**Expected behavior/code**
Link text should be concise and descriptive. Avoid duplicating link text with the title attribute.

**Environment**
Chrome 92, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
4.1.2 Name, Role, Value (Advisory Only)

**Possible Solution**
Remove the `title` attribute from the links.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES

**Comment by mhaack** [comment]
This applies to V1 of the navigation component which is not used any more.
